### ISSUE TITLE
[EPIC] Implement Settings Menu

### ISSUE THREAD

**Issue by amin527** [amin527/pokemon-app#33]
Add a settings menu that can be used to manage application preferences and accessibility options. The settings menu should be designed as an extensible component so additional preferences can be added in future.

## Acceptance Criteria
+ Top navigation contains a button that reads "Settings"
+ When the button is pressed, a drop down menu appears.
+ The drop down menu should contain the following options:
Increased text size options
 Theme selection options
Colour blind accessibility options
Reduced motion options
